src/inflater_managed.rs

```diff
@@ -609,3 +614,10 @@ impl InflaterManaged {
         Ok(())
     }
 }
+
+impl Default for InflaterManaged {
+    #[inline]
+    fn default() -> Self {
```

Comment:
Is InflatorManaged with unknown length default value?

Reply:
If `new()` isn't a reasonable _default_, that's fair. I removed it.